src/test/java/com/marklogic/kafka/connect/source/AbstractIntegrationSourceTest.java

```diff
@@ -89,8 +92,13 @@ void loadSingleAuthorRowIntoMarkLogicWithCustomTime(String uri, String id, Strin
     }
 
     void verifyQueryReturnsFifteenAuthors(List<SourceRecord> sourceRecords, String expectedValue) {
+        verifyQueryReturnsFifteenAuthors(sourceRecords, expectedValue, "null");
```

Comment:
Just do `null` here instead of the string - `"null"` implies to me that it's something besides `null`, but it's just `null`.